script.skin.helper.colorpicker/addon.xml

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+	<addon id="script.skin.helper.colorpicker" name="Skin Helper Service ColorPicker" version="1.0.0" provider-name="marcelveldt">
+	<requires>
+		<import addon="xbmc.python" version="2.13.0"/>
+		<import addon="xbmc.addon" version="12.0.0"/>
+		<import addon="script.module.pil" version="1.1.7"/>
```

Comment:
PIL is used to create the color swatches that can be loaded in a list so skinners can display the picker with their custom colors.

I did this because you can't use colordiffuse with $INFO value inside the itemtemplate